src/main/java/uk/gov/hmcts/ccd/domain/service/stdapi/CallbackInvoker.java

```diff
@@ -147,4 +171,35 @@ private void deduceDataClassificationForNewFields(CaseType caseType, CaseDetails
         jsonNode.ifPresent(value -> data.remove(CALLBACK_RESPONSE_KEY_STATE));
         return jsonNode.flatMap(value -> value.isTextual() ? Optional.of(value.textValue()) : Optional.empty());
     }
+
+    private void validateSignificantItem(AboutToSubmitCallbackResponse response, CallbackResponse callbackResponse) {
+        final SignificantItem significantItem = callbackResponse.getSignificantItem();
+        final List<String> errors = new ArrayList<>();
+
+        if (significantItem != null) {
+            if (significantItem.getType() != SignificantItemType.DOCUMENT) {
+
+                errors.add("Significant Item type incorrect");
+            }
+            if (!URL_VALIDATOR.isValid(significantItem.getUrl())) {
+                errors.add("Url Invalid");
+            }
+            if (isDescriptionEmptyOrNotWithInSpecifiedRange(significantItem)) {
+                errors.add("Description should not be empty but also not more than 64 characters");
+            }
+            if (errors.size() == 0) {
+                response.setSignificantItem(significantItem);
+            } else {
+                callbackResponse.setErrors(errors);
+            }
+        }
+    }
+
+    private boolean isDescriptionEmptyOrNotWithInSpecifiedRange(SignificantItem significantItem) {
+        return StringUtils.isEmpty(significantItem.getDescription())
+            || (StringUtils.isNotEmpty(significantItem.getDescription())
+            && !(significantItem.getDescription().length() > 0
+            && significantItem.getDescription().length() < 65));
+    }
```

Comment:
Can we abstract 0 and 65 into constants.  Easier for testing and maybe they could even be configured